README.md

```diff
@@ -523,6 +523,30 @@ Default: `null`
 
 Type: `string`
 
+#### `mssql_certificates`
+
+A certificate and a key for TLS encryption are created using the `certificate` role.
```

Comment:
```suggestion
Set this variable to generate a certificate and a key for TLS encryption.
The role uses the `fedora.linux_system_roles.certificate` role to generate TLS certificate and key.
```
Replace passive voice with active.
mssql and linux_system_roles are in separate collections and RPMs, so mssql users might not know what we relate to when saying "certificate role". I am using the FQCN to point to roles to make it clearer. For the downstream, I then replace accordingly in the spec file.

Reply:
I think I can edit the README to fix style, e.g. use active voice and make it consistent with the rest, after you finish your work, it will save you some time.